src/control/cmd/dmg/auto.go

```diff
@@ -45,6 +46,11 @@ type configGenCmd struct {
 }
 
 func (cmd *configGenCmd) confGen(ctx context.Context) (*config.Server, error) {
+	// TODO: decide whether we want meaningful JSON output
+	if cmd.JSONOutputEnabled() {
+		return nil, cmd.OutputJSON(nil, errors.New("JSON output not supported"))
+	}
```

Comment:
I think it'd be valuable to have JSON output, if only because it makes tasks easier to automate. The config file text could be a string member inside a struct.

Reply:
This is just migrated code, I won't introduce that as part of this PR if that's okay. Created https://daosio.atlassian.net/browse/DAOS-14121 to address this but I'm not sure in how useful it is as the intended command usages just to pipe stdout to a file.

Reply:
fixed up as I had to re-push for spelling errors, done